x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/execution/search/QlSourceBuilder.java

```diff
@@ -21,6 +22,7 @@
  * the resulting ES document as a field.
  */
 public class QlSourceBuilder {
+    public static final Version FIELDS_API_USAGE_VERSION = Version.V_7_10_0;
```

Comment:
I personally find this naming less evocative ("is the fields api only used in that version?"), but not a biggie.

Reply:
The upside is that these are internal constants that we can rename in the future if we find better names.
Any suggestions for better naming?
`FIELDS_API_MIGRATION_VERSION`, `MIGRATE_TO_FIELD_API_VERSION`, `SWITCH_OVER/TO_FIELDS_API_VERSION` ?